### PR TITLE
Docs: integration guides

### PR DESCRIPTION
**Pull request by MildTomato**
- [x] Prisma: https://docs-git-docs-integration-prisma-supabase.vercel.app/docs/guides/integrations/prisma
- [x] [Auth0](https://auth0.com/blog/using-nextjs-and-auth0-with-supabase/)
- [x] Vercel

http://localhost:3006/docs/guides/integrations/prisma

Screenshot 2021-08-31 at 19 59 42

### CODE REVIEW

**Comment by dthyresson** [comment]
Hi @MildTomato we use Prisma in RedwoodJS and I have a PR in for its documentation to describe how to setup Supabase with connection pooling and pgBouncer: https://github.com/redwoodjs/redwoodjs.com/pull/785

```
We recommend that you connect to your Supabase Postgres instance using SSL which you can do by setting `sslmode` to `require` on the connection string:

 ```
 // not pooled typically uses port 5432
 postgresql://postgres:mydb.supabase.co:5432/postgres?sslmode=require
 // pooled typically uses port 6543
 postgresql://postgres:mydb.supabase.co:6543/postgres?sslmode=require&pgbouncer=true
 ```

It's important in RedwoodJS if connecting via Prisma in a serverless function to use connection pooling as sell as ssl.

Prisma has a specific parameter to tell it that it needs to support pgBouncer:

https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer

The important note about using Prisma with Supabase is in the way it handles migrations -- meaning, it cannot migrate the same way when connecting to the pool than not:

https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer#prisma-migrate-and-pgbouncer-workaround

The workaround is that when migrating, one uses the non-pooled connection -- but when running the client one should use the pooled connection.

**Comment by MildTomato** [comment]
sorry, this got fogotten about - will need to jump back on this

File: web/sidebars.js [hunk 1]
```diff
@@ -182,6 +182,16 @@ module.exports = {
...
       ],
     },
+    {
+      type: 'category',
+      label: 'Integrations',
+      collapsed: false,
+      items: [
+        'guides/integrations/prisma',
+        'guides/integrations/auth0',
+        'guides/integrations/draftbit',
```
Comment (kiwicopple) [inline web/sidebars.js:192]: - add vercel here
- make them alphabetical order

Comment (dijonmusters) [reply web/sidebars.js:191]: Good spot! Fixed and pushed 👍

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**